aten/src/ATen/cpu/vec/vec512/vec512_float.h

```diff
@@ -121,6 +121,29 @@ template <> class Vectorized<float> {
       std::memcpy(ptr, tmp_values, count * sizeof(float));
     }
   }
+  void store_float_as_uint8(uint8_t* dst_data) {
```

Comment:
It might be safe to do saturation and clamp as a general util function but seems not an efficient choice for inductor codegen if the code pattern is most likely quantization which has saturation and clamp before this particular intrinsics is called, therefore the logic here seems duplicate. It's probably ok to start with this slow path but can you explore the fast path too? Maybe we can match the pattern of quantization op in the loop body and invoke the fast path without explicit saturation and clamp?

Reply:
Thanks for the comment. Agree and I have marked it as a TODO and will optimize it in the following up PR after some init study of the new frontend API.